src/eval.c

```diff
@@ -1122,6 +1122,7 @@ get_lval(
 		    return NULL;
 		lp->ll_name_end = tp;
 	    }
+	    // TODO: check inside class *****************************
```

Comment:
The `cctx` need to be passed to `get_lval` to detect whether this is compiled from inside a class.